data/internal/Rogue_Mage/units/Bandit_Rogue_Mage.cfg

```diff
@@ -1,31 +1,28 @@
-#textdomain wesnoth-tdg
+#textdomain wesnoth-units
 
 [unit_type]
     id=Rogue Mage
     name= _ "Rogue Mage"
     race=human
     gender=male,female
-    image="units/rogue-mage/rogue-mage.png"
-    #     profile="portraits/humans/ruffian.webp"
+    image="units/rogue-mage.png"
     hitpoints=26
-    movement_type=elusivefoot
+    movement_type=smallfoot
     movement=5
-    experience=46
+    experience=54
     level=1
-    alignment=liminal
-    advances_to=Shadow Mage,Arcane Enforcer
+    alignment=chaotic
+    advances_to=Arcane Enforcer,Shadow Mage
     cost=19
     usage=mixed fighter
-    description= _ "Not every student has the skill and temperament to become a mage. Novices often find themselves abandoned by their masters, sometimes for attempting to practice forbidden arts, but more often for mundane transgressions such as theft or embezzlement, or even a simple lack of funds.
+    description= _ "Not every student has the skill and temperment to become a mage. Novices often find themselves abandoned by their masters, sometimes for attempting to practice forbidden arts, but more often for mundane transgressions such as theft or embezzlement, or even a simple lack of funds.
 
-While most of these failed wizards switch to other paths, a driven few turn to banditry as a way of continuing to fund their independent studies. Now completely outside the law, Rogue Mages do whatever is necessary to support themselves, and will often dabble in the use of black magic as a result. Although they are rarely as skilled as wizards with more formal training, their spells can still be quite dangerous, and their banditry has resulted in moderate skill with the short sword."
-    {DEFENSE_ANIM "units/rogue-mage/rogue-mage-defend2.png" "units/rogue-mage/rogue-mage-defend1.png" {SOUND_LIST:HUMAN_OLD_HIT} }
+While most of these failed wizards switch to other paths, a driven few turn to banditry as a way of continuing to fund their independent studies. Now completely outside the law, Rogue Mages do whatever is necessary to support themselves, and will often dabble in the use of black magic as a result. Although they are rarely as skilled as wizards with more formal training, their spells can still be quite lethal, and their banditry has resulted in moderate skill with the short sword."
```

Comment:
I prefer the original. Either revert to 'quite dangerous' or change to 'lethal'. I don't consider 'lethal' to be a comparative term, same as with the word 'unique', but I notice some, particularly in en_US, treat it as such.